### DISCUSSION TITLE
Compatibility with LoRa RN2483 module.

### DISCUSSION THREAD

**Discussion by raghavyadav11** [sandeepmistry/arduino-LoRa#643] [General]
Hey,
I am new to LoRa and I am working with TheThingsUno board with inbuilt LoRa RN2483 module. I want to create a p2p network using this node.
I want to know if I can use this library for this purpose? If not, can anyone please suggest any alternative?

Thanks in advance.

**Comment by Kongduino** [comment]
No. The RN2483 is a UART-controlled module, so this library won't work with it. Besides, the TheThingsUno is way too underpowered to do anything but a basic PING/PONG test.
